code/modules/surgery/surgery_step.dm

```diff
@@ -23,7 +23,7 @@
 		if(!tool)
 			success = TRUE
 		if(iscyborg(user))
-			success = TRUE	
+			success = TRUE
```

Comment:
Whitespace removal. No functional change here.

Reply:
Pretty sure vscode just did that when i saved?

Reply:
Just making a note is all.